test/e2e/utils/nodetopology/nodetopology.go

```diff
@@ -68,14 +69,20 @@ func IsValidNodeTopology(nodeTopology *v1alpha1.NodeResourceTopology, tmPolicy s
 		foundNodes++
 
 		if !IsValidCostList(zone.Name, zone.Costs) {
+			framework.Logf("invalid cost list for %q %q", nodeTopology.Name, zone.Name)
 			return false
 		}
 
 		if !IsValidResourceList(zone.Name, zone.Resources) {
+			framework.Logf("invalid resource list for %q %q", nodeTopology.Name, zone.Name)
 			return false
 		}
 	}
-	return foundNodes > 0
+	ret := foundNodes > 0
+	if !ret {
+		framework.Logf("invalid nodes for %q", nodeTopology.Name)
```

Comment:
Maybe I am being pedantic here I think we can improve this log message. Here no nodes are found for the NRT name so might be worth tweaking the log a little bit as this gives the impression that we got a node that wasn't valid.

Reply:
good point, will reword